### ISSUE TITLE
Insufficient contrasts

### ISSUE THREAD

**Issue by ffoodd** [Orange-OpenSource/Orange-Boosted-Bootstrap#34]
While working with Boosted v4 alpha 5, aXe (through gulp-axe-webdriver) warns me about (at least) three components with insufficient contrast ratios:

- active item in breadcrumb
- primary button
- disabled custom checkboxes' label

In fact those 3 components would pass if we accepted a 3:1 ratio, so to me that's not really an issue; but the fact is Orange a11y-guidelines requires 4.5:1 and 7:1.

So I'm wondering:

- should I try to configure aXe to accept 3:1 ratio?
- should we fix Boosted to pass 4.5:1 ratio?
- should a11y-guidelines be less restrictive?

FWIW, the primary orange color just pass 3.01:1 on a white :/ Seems a bit odd to require 4.5:1 when working on internal project…

For now I exclude those components from aXe testing:
```
exclude: '.breadcrumb,.btn-primary,[disabled] ~ .custom-control-description'
```
but I feel kinda dirty ><

**Comment by Lausselloic** [comment]
Hello,

Yes we know about it. After many discussion with Orange Brand considering the contrast between the core color orange (initially #F60) and white background or foreground, we have tatus on the #F16E00 orange color to provide the best contrast, even if it's not today enough to be really full wcag compliant.

So we are working on aXe, to reject contrast-error on the core color only.

Sorry for that, but at time, we've no better solution, we're trying to limit usage of orange color with the white but it's in our company ADN

**Comment by ffoodd** [comment]
I agree, thanks :) 
Let me know when you'll find anything within aXe to help on this!
